src/Forecast.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from "react";
+
+const api = {
+    key: "97f6c11a941a6c4ff34b1c26885988e1",
```

Comment:
Just a word of warning when it comes to API keys - they should not be stored in plain text in the code like this in the real world. Usually we use stuff like google secrets/vault/special config files which we store these kind of keys in.
For this project its ok, but just something to bear in mind :)